curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-data-structures-by-building-a-shopping-cart/63f034d012f74627ce538d3a.md

```diff
@@ -0,0 +1,357 @@
+---
+id: 63f034d012f74627ce538d3a
+title: Step 40
+challengeType: 0
+dashedName: step-40
+---
+
+# --description--
+
+Finally, update the `display` property of the `style` object of the `cartContainer` variable to another ternary which checks if `isCartShowing` is true. If it is, set the `display` property to `block`, otherwise set it to `none`.
```

Comment:
Finally, update the `display` property of the `style` object of the `cartContainer` variable to another ternary which checks if `isCartShowing` is true. If it is, set the `display` property to `block`, otherwise, set it to `none`.

Reply:
`otherwise` should be preceded with a comma but not followed by a comma: https://ahdictionary.com/word/search.html?q=otherwise

Reply:
Okay. This PR has my approval.